terraform/cloudwatch.tf.json.template.py

```diff
@@ -173,6 +190,32 @@ def prod_qualified_resource_name(name: str) -> str:
                     }
                     for lambda_ in config.lambda_names()
                 ),
+                *(
+                    {
+                        'aws_cloudwatch_metric_alarm': {
+                            alarm_name: {
+                                'alarm_name': config.qualified_resource_name(alarm_name, suffix='.alarm'),
+                                'namespace': 'AWS/Lambda',
+                                'dimensions': {
+                                    'FunctionName': f'${{aws_lambda_function.{threshold.lambda_name}.function_name}}'
+                                },
+                                'metric_name': threshold.metric.capitalize(),
```

Comment:
I find it a bit concerning that we wait this long to capitalize the metric name. I think it would be better for the `MetricThreshold` instances to store the correct metric name throughout their lifespan, rather than relying on users of the class to perform the conversion when they need to use the attribute. In other words, I would move the call to `capitalize` to where the `MetricThreshold` constructors are called.

Reply:
Metric names were capitalized elsewhere, but the call to `.capitalize()` was not removed